src/components/DescContactInfo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+
+const DescContactInfo = () => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.textInstructions}>
+        Esta información servirá para ponernos en contacto contigo más rapido
+        cuando sea necesario
```

Comment:
Add a constant  folder into `src` and create a constants file import the text as variable